### PR TITLE
refactor: Enhance code quality for stricter style checks

### LINKED ISSUE #150: nickczh - stricter styling checks

### PR DESCRIPTION
**Pull request by nickczh**
# Checklist for feature implementation and modification
Closes #150 
- [x] Added relevant milestone (under milestone tab on the right)
- Implementation
  - [ ] Sources (if any) credited under both relevant section(s) and Developer Guide (Acknowledgement)
  - [ ] JUnit Test(s)
- Documentation
  - [ ] JavaDoc for classes and methods
  - Developer Guide
    - [ ] Design (Can add as a todo statement for later on)
    - [ ] Implementation (Can add as a todo statement for later on)
    - [ ] Relevant User Story present
    - [ ] Non-functional requirement(s) that the feature addresses (if any)
    - [ ] Glossary of the (technical) term(s) used (if any)
    - [ ] Specialised instructions for manual testing (if any)
  - User Guide
    - [ ] Description and usage of feature (if any)
  - Contributions (under `/docs/team/*.md`)
    - [ ] Code/Feature/Enhancement contribution
    - [ ] Developer Guide contribution
    - [ ] User Guide contribution
    - [ ] Misc and team-based admin tasks (if any)

# Checklist for non-implementation related PR
Closes #<Issue_Number>
- [ ] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [ ] Developer Guide contributions (if any)
  - [ ] User Guide contributions (if any)
  - [ ] Review contributions (if any)
  - [ ] Miscellaneous and team-based admin tasks (if any)

# Additional Information

Nil

### CODE REVIEW

File: src/test/java/meditracker/command/AddCommandTest.java [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
 import meditracker.exception.HelpInvokedException;
+import meditracker.exception.MediTrackerException;
 import meditracker.medication.Medication;
 import meditracker.medication.MedicationManager;
 import meditracker.medication.MedicationManagerTest;
 import meditracker.time.MediTrackerTime;
+
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
```
Comment (T0nyLin) [inline src/test/java/meditracker/command/AddCommandTest.java:16]: Hey Nick, I believe lines 14 to 16 are in the wrong order

Please refer to this: https://se-education.org/guides/tutorials/intellijCodeStyle.html#tweak-import-order

**Review by T0nyLin — CHANGES_REQUESTED** [review]
Thanks for making the tedious changes with the checkstyle. Just a small import ordering change, please take a look

File: src/test/java/meditracker/command/AddCommandTest.java (hunk 1 above)
Comment (nickczh) [reply src/test/java/meditracker/command/AddCommandTest.java]: Thanks for the great catch! I have made the changes in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/248/commits/0d8b42ee2fce4968c4dc00220668a3ced9147fa4. Please help me verify :)

**Review by T0nyLin — APPROVED** [review]
LGTM!

**Pull request merged**